.github/workflows/ci.yml

```diff
@@ -167,12 +173,13 @@ jobs:
   linux-build-native-latest:
     name: PR - Linux - Native build - Latest Version
     runs-on: ubuntu-latest
-    needs: detect-test-suite-modules
+    needs: prepare-jvm-native-latest-modules-mvn-param
     env:
       MODULES_ARG: ${{ needs.detect-test-suite-modules.outputs.MODULES_ARG }}
```

Comment:
I think `MODULES_ARG` can be removed now

Reply:
Thanks missed this! Updated.